src/stores/game.js

```diff
@@ -11,7 +11,7 @@ function queenCount(player) {
   if (!player) {
     return null;
   }
-  return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
+  return player?.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0) ?? 0;
```

Comment:
It might be worth doing the player check first instead of combining it all for clarity. Easy to lose when trailing the reducer.

```
if (!player) {
    return 0;
}

return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
```

Reply:
lol yep